test/integration/cloudresource_e2e_test.go

```diff
@@ -198,6 +199,29 @@ var _ = Describe(fmt.Sprintf("%s,%s: NetworkPolicy On Cloud Resources", focusAws
 			}
 			err := utils.ConfigureK8s(kubeCtl, defaultANPParameters, k8stemplates.DefaultANPSetup, false)
 			Expect(err).ToNot(HaveOccurred(), "failed to add default ANP rule")
+
+			// Check antrea-agent has connected to antrea-controller.
```

Comment:
Put this in new function

Reply:
Will do it in next version and update the PR

Reply:
Done.